### ISSUE TITLE
Failed to build cuSPARSELt examples with cuSPARSELt 0.3.0.3

### ISSUE THREAD

**Issue by jeng1220** [NVIDIA/CUDALibrarySamples#101]
error message:
```cpp
$ make

nvcc --std=c++14  -I/home/scratch.svc_compute_arch/release/cuda_toolkit/r11.8/x86_64/11.8.0.016/include -I/home/rjeng/workspace/nv/libcusparse_lt-linux-x86_64-0.3.0.3-archive/include  spmma2_example.cpp -o spmma2_example \
         -L/home/rjeng/workspace/nv/libcusparse_lt-linux-x86_64-0.3.0.3-archive/lib64 -lcusparseLt -lcudart -lcusparse -ldl
spmma2_example.cpp: In function ‘int main()’:
spmma2_example.cpp:278:79: error: too few arguments to function ‘cusparseStatus_t cusparseLtMatmulPlanInit(const cusparseLtHandle_t*, cusparseLtMatmulPlan_t*, const cusparseLtMatmulDescriptor_t*, const cusparseLtMatmulAlgSelection_t*, size_t)’
     CHECK_CUSPARSE( cusparseLtMatmulPlanInit(&handle, &plan, &matmul, &alg_sel) )
                                                                               ^
spmma2_example.cpp:67:32: note: in definition of macro ‘CHECK_CUSPARSE’
     cusparseStatus_t status = (func);                                          \
                                ^~~~
In file included from spmma2_example.cpp:53:0:
/home/rjeng/workspace/nv/libcusparse_lt-linux-x86_64-0.3.0.3-archive/include/cusparseLt.h:238:1: note: declared here
 cusparseLtMatmulPlanInit(const cusparseLtHandle_t*             handle,
 ^~~~~~~~~~~~~~~~~~~~~~~~
spmma2_example.cpp:328:79: error: too few arguments to function ‘cusparseStatus_t cusparseLtMatmulPlanInit(const cusparseLtHandle_t*, cusparseLtMatmulPlan_t*, const cusparseLtMatmulDescriptor_t*, const cusparseLtMatmulAlgSelection_t*, size_t)’
     CHECK_CUSPARSE( cusparseLtMatmulPlanInit(&handle, &plan, &matmul, &alg_sel) )
                                                                               ^
spmma2_example.cpp:67:32: note: in definition of macro ‘CHECK_CUSPARSE’
     cusparseStatus_t status = (func);                                          \
                                ^~~~
In file included from spmma2_example.cpp:53:0:
/home/rjeng/workspace/nv/libcusparse_lt-linux-x86_64-0.3.0.3-archive/include/cusparseLt.h:238:1: note: declared here
 cusparseLtMatmulPlanInit(const cusparseLtHandle_t*             handle,
 ^~~~~~~~~~~~~~~~~~~~~~~~
Makefile:64: recipe for target 'spmma2_example' failed
make: *** [spmma2_example] Error 1
```

**Comment by qanhpham** [comment]
Hi @jeng1220. There have been updates in cusparseLt APIs recently and you're using old header file _cusparseLt.h_.
Please check if you have the latest header file. If the file isn't updated in install package, we'll fix it soon.
Meanwhile, a quick fix for this issue is editing the function signature in _cusparseLt.h_ as:

```
 cusparseLtMatmulPlanInit(const cusparseLtHandle_t*             handle,
                          cusparseLtMatmulPlan_t*               plan,
                          const cusparseLtMatmulDescriptor_t*   matmulDescr,
                          const cusparseLtMatmulAlgSelection_t* algSelection);
```

Let me know if you have other problems.

**Comment by fbusato** [comment]
Fixed in 92e305dc282a10b3e82285bf7c8b9210243c80c7
